code/datums/uplink_items/uplink_general.dm

```diff
@@ -503,6 +503,13 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	item = /obj/item/clothing/glasses/chameleon/thermal
 	cost = 15
 
+/datum/uplink_item/stealthy_tools/thermal
```

Comment:
why is this under thermals? 
should really be pathed `...stealthy_tools/nightvision`